### PR TITLE
ZIOS-9956: Adapted UI for new logging System APIs

### PR DESCRIPTION
**Pull request by nicorsm**
### Issues

PR to adapt UI project to https://github.com/wireapp/wire-ios-system/pull/21 (avoid errors to be logged on production builds).

### Solutions

Main changes are:

- I'm not passing the size of the circular array, since we're not using it anymore
- I'm passing the `isInternal` flag, so we can detect if the build is internal or production - it's also useful for writing tests.

## Dependencies

Needs releases with:

- [x] https://github.com/wireapp/wire-ios-system/pull/21

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/SelfProfile/SelfProfileViewController.swift [hunk 1]
```diff
@@ -126,11 +126,25 @@ final internal class SelfProfileViewController: UIViewController {
     }
     
     private func createConstraints() {
-        constrain(view, accountSelectorController.view, profileContainerView) { selfView, accountSelectorControllerView, profileContainerView in
+        var selfViewTopMargin: CGFloat = 12
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/SelfProfile/SelfProfileViewController.swift]: Seem to contain some unrelated changes, merging consequence?

Comment (nicorsm) [reply Wire-iOS/Sources/UserInterface/SelfProfile/SelfProfileViewController.swift]: Yes, I guess it's because of this one: https://github.com/wireapp/wire-ios/pull/2066/commits/098b184484cb354dd8e83459ffea67d4292b9182

**Comment by nicorsm** [comment]
Just checked tests that are working fine locally. I'll merge the branch.
